baybe/utils/numeric.py

```diff
@@ -2,12 +2,12 @@
 from typing import List
 
 import numpy as np
```

Comment:
General question here: Should we also re-organize the numpy import in the same way the torch imports are re-organized? At least for consistency, I think it would be nicer to have  something like `from torch import float64 as t_float64` and `from numpy import float64 as n_float64`.

Reply:
Hey, sorry to interrupt, but I think this is a step in the wrong direction. I just wanted to briefly flag it here so that you don't make more changes of this kind until we have aligned, to save you some potential unnecessary work. Let me give the full picture in my review that is coming in a few mins 😃

Reply:
Since we agreed to a different approach in general, I mark this conversation as resolved